code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -300,7 +300,7 @@
 	spawn_flags = SPECIES_CAN_JOIN
 
 /datum/species/diona
-	spawn_flags = SPECIES_CAN_JOIN
+	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED
```

Comment:
If these tags are the same in default station.dm, then this whole section can be removed.